cmd/create/operatorroles/by_prefix.go

```diff
@@ -304,6 +304,7 @@ func createRolesByPrefix(r *rosa.Runtime, prefix string, permissionsBoundary str
 		}
 
 		var policyArn string
```

Comment:
Considering this only exists now to do [this](https://github.com/openshift/rosa/blob/71c1af575bf4fbd6d88610a8201cf2402df132bd/cmd/create/operatorroles/by_prefix.go#L405), can't it also be included in the new slice of policy arns?